rosmon_core/src/main.cpp

```diff
@@ -75,6 +75,9 @@ void usage()
 		"                 Use GROUP as name of the launch group. By default, empty\n"
 		"  --launch-config=CONFIG\n"
 		"                 Use CONFIG as name of the launch config. By default, empty\n"
+                "  --respawn-attr=force_true|force_false\n"
```

Comment:
as mentioned before lets keep an option to `obey` the respawn rules mentioned in launch file which will be default.